Automattic/sensei
# Replace customize links

Fixes #5831,#5832

### Changes proposed in this Pull Request

* Updates all the 'Customize' links to point to the Site editor.
* In Courses, removes the customize link when it is enabled globally.
* I kept the old implementation that links to the customizer as we might need it in the future.
* Considered to add a 'Customize colors'. Probably should be done as part of anohter PR
* Updates the images in template settings.

### Testing instructions
- Use the 'Customize' links in Appeareance settings and observe that they redirect to the Site Editor.
- Use the links in the Learning Mode panel in a course and observe that they redirect to Sensei settings.
